Modules/Synchronization/src/Synchronization.Application/ApplicationErrors.cs

```diff
@@ -65,5 +65,10 @@ public static ApplicationError UnexpectedSyncRunType(SyncRun.SyncRunType expecte
         {
             return new ApplicationError("error.platform.validation.syncRun.unexpectedSyncRunType", $"The current operation only supports sync runs of type '{expectedType}'.");
         }
+
+        public static ApplicationError CannotStartSyncRunWhileIdentityIsToBeDeleted()
+        {
+            return new ApplicationError("error.platform.validation.syncRun.cannotStartSyncRunWhileIdentityIsToBeDeleted", "Cannot start a sync run. Identity is marked for deletion.");
+        }
```

Comment:
I would use something like: "Sync run cannot be started when identity is marked for deletion."

Reply:
Changes made.